frontend/src/console/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,287 @@
+                  <h2>
+                    <WrappedMessage id="heroText" messages={messages} />
+                  </h2>
+                </Col>
+              </Row>
+              <Row>
+                <Col md={6}>
+                  <TextInputField
+                    fieldName="title"
+                    messages={messages}
+                    value={this.state.title}
+                    onBlur={this.updateHeroText}
+                    onChange={this.onChange}
+                  />
+                </Col>
+                <Col md={6}>
+                  <TextInputField
+                    fieldName="subtitle"
+                    messages={messages}
+                    value={this.state.subtitle}
+                    onBlur={this.updateHeroText}
+                    onChange={this.onChange}
+                  />
+                </Col>
+              </Row>
+              <Row className="hero-customizations">
+                <Col md={4}>
+                  <ImageUploadField
+                    // prettier-ignore
+                    customUploadMessage={(
+                      <WrappedMessage id="uploadHeroCoverImage" messages={messages} />
+                  )}
+                    updateImage={(image: File) => {
+                      this.updateImage('heroCoverImage', image);
+                    }}
+                    clearError={() => {
+                      this.props.clearErrorMessage('heroCover');
+                    }}
+                  />
+                  {this.activeInstanceDataExists() &&
+                    instance.data!.heroCoverImage && (
+                      <button
+                        className="reset-value"
+                        type="button"
+                        onClick={() => {
+                          this.removeImage('heroCoverImage');
+                        }}
+                      >
+                        Remove
+                      </button>
+                    )}
+                </Col>
+                {themeData &&
+                  themeData.version === 1 && [
+                    <Col md={4} key="heroTitleColor">
+                      <ColorInputField
+                        fieldName="homePageHeroTitleColor"
+                        initialValue={themeData.homePageHeroTitleColor}
+                        onChange={this.onChangeColor}
+                        messages={messages}
+                        loading={instance.loading.includes('draftThemeConfig')}
+                      />
+                    </Col>,
+                    <Col md={4} key="heroSubtitleColor">
+                      <ColorInputField
+                        fieldName="homePageHeroSubtitleColor"
+                        initialValue={themeData.homePageHeroSubtitleColor}
+                        onChange={this.onChangeColor}
+                        messages={messages}
+                        loading={instance.loading.includes('draftThemeConfig')}
+                      />
```

Comment:
The styling of these fields doesn't look correct:
![image](https://user-images.githubusercontent.com/3189670/78460627-a0b26400-76c2-11ea-956d-148e6b8d25b9.png)

Reply:
@Agrendalath, can you give more details on what doesn't look correct?

Reply:
@lgp171188, ah, I should have been more precise here - the `Hero/styless.scss` is missing `background-color: white;` under `.form-control`.